### DISCUSSION TITLE
Has anyone encountered act(...) warnings here?

### DISCUSSION THREAD

**Discussion by thkim1011** [testing-library/user-event#1200] [Q&A]
Since `user.type` gets executed in [`asyncWrapper`](https://github.com/testing-library/react-testing-library/blob/edb6344d578a8c224daf0cd6e2984f36cc6e8d86/src/pure.js#L37-L59), `user.type` executes the following line with `IS_REACT_ACT_ENVIRONMENT` as false.

https://github.com/testing-library/user-event/blob/d0362796a33c2d39713998f82ae309020c37b385/src/utility/type.ts#L40

This is causing the following warning to pop up.
```
  console.error
    Warning: The current testing environment is not configured to support act(...)

      176 |   }
      177 |
    > 178 |   consoleError(str, ...args);
          |   ^
      179 |   return;
      180 | });
      181 |
```
However, for some other tests, I don't see this happening, so I'm assuming that the `setState` call shouldn't have happened until `user.type` finished? Has anyone seen this before?

**Comment by lionskape** [comment]
Facing with the same issue.
I found that if you have only one use event call in test - there is no warning
